common/test/acceptance/pages/lms/progress.py

```diff
@@ -115,7 +115,7 @@ def _chapter_index(self, title):
         Return the CSS index of the chapter with `title`.
         Returns `None` if it cannot find such a chapter.
         """
-        chapter_css = '.chapters section .hd'
+        chapter_css = '.chapters>section h3'
```

Comment:
So that's why it was passing before - the timing/test ordering never caused a subset/superset mismatch, and something in the recent changes perturbed that fragile coincidence to the point of failing sporadically.